lib/ghomperm.gi

```diff
@@ -1442,7 +1447,25 @@ local   D,  I,G;
     SetStabChainMutable(G,I);
     return G;
   fi;
-end );
+end;
+
+if IsBound(HPCGAP) then
+# In HPC-GAP we can't filter on HasStabChainMutable because it is thread-local
+    InstallMethod( ImagesSet,"constituent homomorphism",
+        CollFamSourceEqFamElms, [ IsConstituentHomomorphism, IsPermGroup], 1,
```

Comment:
Perhaps put the `1` on its own line, and add a comment like "# adjust rank for the missing HasStabChainMutable" ?